Bottom: 699f655f96c3c3216140fbb8413ce49b11a841d3
Top:    aca5ed0f7c6c8b31c7626f92e72dfef531d38271
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 23:13:20 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index d854b0b..8c737a3 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2610,7 +2610,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
